src/Umbraco.Web.UI.Client/src/views/components/content/umb-content-node-info.html

```diff
@@ -19,6 +19,22 @@
                 </ul>
             </umb-box-content>
         </umb-box>
+        <umb-box data-element="node-info-redirects">
+            <umb-box-header title-key="redirectUrls_redirectUrlManagement"></umb-box-header>
+            <umb-box-content class="block-form">
+                <div style="position: relative;">
+                    <div ng-if="loadingRedirectUrls" style="background: rgba(255, 255, 255, 0.8); position: absolute; top: 0; left: 0; right: 0; bottom: 0;"></div>
+                    <umb-load-indicator ng-if="loadingRedirectUrls"></umb-load-indicator>
+                    <ul class="nav nav-stacked" style="margin-bottom: 0;" ng-show="hasRedirects">
+                        <li ng-repeat="redirectUrl in redirectUrls">
+                            <a href="{{redirectUrl.originalUrl}}"><i ng-class="value.icon" class="icon-out"></i> {{redirectUrl.originalUrl}}</a>
+                        </li>
+                    </ul>
+                    <div ng-show="!hasRedirects && !loadingRedirectUrls && !urlTrackerDisabled"><localize key="redirectUrls_noRedirects"></localize></div>
+                    <div ng-show="urlTrackerDisabled">Redirect Url Management is disabled</div>
```

Comment:
Instead of showing: we can't show you information, I think it would be better to not show this panel altogether.

Reply:
Because it's only clear after the api request has pulled back the redirects, whether there is anything to show, i was getting an annoying wobble when switching to that tab, eg you'd see the redirects panel for a second, then it would either disappear, or appear, with a flicker, I've added ng-cloak and an inline style of display:none, and I think the flicker has gone! - but obviously this probably isn't the 'bestest' front end way of achieve this... but I would take advice on how to do it super neatly?